test/EFCore.PG.FunctionalTests/Query/ManyToManyQueryNpgsqlFixture.cs

```diff
@@ -7,5 +10,210 @@ namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
     public class ManyToManyQueryNpgsqlFixture : ManyToManyQueryRelationalFixture
     {
         protected override ITestStoreFactory TestStoreFactory => NpgsqlTestStoreFactory.Instance;
+
+        // We default to mapping DateTime to 'timestamp with time zone', but the seeding data has Unspecified DateTimes which aren't
+        // supported. So we need to explicitly set some column types to 'timestamp without time zone', but this is difficult/problematic
+        // for some of the many-to-many join entities configured below. So for now we duplicate the entire method.
+        // TODO: Clean this up
```

Comment:
https://github.com/dotnet/efcore/issues/26068

Reply:
Yep thanks, the EF tests were a pain here.